docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "Manage an Aurora Serverless v2 cluster with the ACK RDS Controller"
+description: "Create an Aurora Serverless v2 cluster from an Amazon Elastic Kubernetes Service (EKS) deployment."
+lead: "Create and manage an Aurora Serverless v2 cluster directly from Kubernetes"
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 45
+toc: true
+---
+
+In this tutorial you will learn how to create and manage
+[Aurora Serverless v2](https://aws.amazon.com/rds/aurora/serverless/) instances
+from an Amazon Elastic Kubernetes (EKS) deployment.
+
+## Prerequisites
+
+There are two ways to create the environment for running this tutorial.
+
+The first method uses [Amazon EKS Workshop](https://www.eksworkshop.com/010_introduction/). The second method involves deploying your own Amazon EKS cluster.
+
+### Method 1: Use Amazon EKS Workshop
+
+1. Create a [workspace](https://www.eksworkshop.com/020_prerequisites/workspace/)
+2. Install the [Kubernetes tools](https://www.eksworkshop.com/020_prerequisites/k8stools/)
+3. Create an [IAM role for workspace](https://www.eksworkshop.com/020_prerequisites/iamrole/)
+4. Attach [IAM role to workspace](https://www.eksworkshop.com/020_prerequisites/ec2instance/)
+5. Update [IAM settings](https://www.eksworkshop.com/020_prerequisites/workspaceiam/)
+6. [Create KMS custom managed keys](https://www.eksworkshop.com/020_prerequisites/kmskey/)
+
+### Method 2: Deploy an Amazon EKS cluster
+
+1. Create an Amazon EKS cluster
+2. Install [eksctl tools](https://www.eksworkshop.com/030_eksctl/prerequisites/)
+3. Install [HELM](https://www.eksworkshop.com/beginner/060_helm/helm_intro/install/)
+4. Install [EKS](https://www.eksworkshop.com/030_eksctl/launcheks/)
+5. Test [EKS cluster](https://www.eksworkshop.com/030_eksctl/test/)
+6. (Optional) [Grant console access to EKS cluster](https://www.eksworkshop.com/030_eksctl/console/#:~:text=The%20EKS%20console%20allows%20you,granted%20permission%20within%20the%20cluster.)
```

Comment:
What about `kubectl`? It seems that these are mainly covered under the `Kubernetes tools` that the previous installation method suggests - https://www.eksworkshop.com/020_prerequisites/k8stools/

Reply:
@jkatz I don't think it is correct two say, two method to install, both are required.

Reply:
Changed this to "Set up Amazon EKS Workshop" and "Deploy ..."